app/src/main/java/android/app/lotus/view/auth/Login.kt

```diff
@@ -36,21 +35,11 @@ import androidx.compose.ui.unit.sp
 import androidx.navigation.NavHostController
 
 @Composable
-fun Login(navController: NavHostController, authViewModel: AuthViewModel) {
+fun Login(navController: NavHostController, mainViewModel: MainViewModel) {
     Box (
         modifier = Modifier.fillMaxSize(),
         contentAlignment = Alignment.BottomCenter
     ) {
-
-        Image(
```

Comment:
Why remove the image?

Reply:
When switching to dark mode the image wasn't very visible (due to there being dark parts in the image). We can find a better image and use it instead in the future ;)